Lib/fontTools/ttLib/ttGlyphSet.py

```diff
@@ -134,11 +136,15 @@ def keys(self):
 
 	def has_key(self, glyphName):
 		return glyphName in self._ttFont['glyf']
+
 	__contains__ = has_key
 
 	def __getitem__(self, glyphName):
 		return _TTVarGlyphGlyf(self._ttFont, glyphName, self.location)
 
+	def __len__(self):
+		return len(self._ttFont['glyf'])
+
```

Comment:
You can assign `self._ttFont['glyf']` as `self._glyphs` and remove a few of the methods now.

Reply:
done, thanks